### ISSUE TITLE
[ATM9 tts] EMC not showing up in AE2 system

### ISSUE THREAD

**Issue by CosmoPoptart** [62832/AppliedE#22]
I have a transmutation module in my system with an open channel, I have about 500k EMC but for some reason it just is not showing up in my system

**Comment by 62832** [comment]
Did you try replacing the module?

**Comment by CosmoPoptart** [comment]
Yes it is connected properly and I have tried replacing it I have a video of the problem if you would like to see that but I cant send it through github, It only accepts images

**Comment by 62832** [comment]
Sure, send a link and I can have a look at the video.

**Comment by CosmoPoptart** [comment]
Here is the video, I have also heard that the module needs to be on a storage bus but I have not tried that yet because it worked fine on my old ME system (https://youtu.be/qqZ-RIo7eFM)

**Comment by 62832** [comment]
Nothing about the setup in that video looks out of the ordinary (and you shouldn't need a storage bus for anything in this situation, I don't know who told you or suggested that), so it is strange that none of your EMC is showing up. I might need to investigate further.

**Comment by 62832** [comment]
However, unless I was mistaken, I did notice that you had autocrafting patterns available for things like cobblestone and XyCraft crystals. I assume those might have come from the module, so what happens if you try to use them to autocraft those things out of EMC?

**Comment by superbob** [comment]
I have the same issue, do you have any idea to fix this?

**Comment by 62832** [comment]
Not really, still. If the OP had replied to me all that time ago to what I last asked on this issue then I probably would have some idea.

**Comment by superbob** [comment]
Ok, I could fix the issue by removing and redoing the setup. It didn't work the first time, but after doing it multiple times at succeeded at some point 🤷‍♂ .
After closing and reoping the save I had the pb again, so again I removed the transmutation setup and put it it again and it worked.
